OracleApplicationExpress/dockerfiles/setup_apex.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Source the properties file
+source ../../config.properties
+
+echo "Removing any existing APEX setup..."
+cd /opt/oracle/apex
+sqlplus -s $DB_USER/$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_SERVICE as sysdba <<EOF
+   alter session set container = PDB1;
+   @apxremov.sql
+   exit;
+EOF
+echo "Finished removing APEX setup..."
+
+# Configure APEX
+echo "Starting APEX setup..."
+cd /opt/oracle/apex
+
+sqlplus -s $DB_USER/$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_SERVICE as sysdba <<EOF
+   alter session set container = PDB1;
+   @apexins.sql SYSAUX SYSAUX TEMP /i/
+   exit;
+EOF
+echo "Finished APEX setup..."
+
+
+
+# Check if APEX_PUBLIC_USER exists before attempting operations
+sqlplus -s $DB_USER/$DB_PASSWORD@$DB_HOST:$DB_PORT/$DB_SERVICE as sysdba <<EOF
+   alter session set container = pdb1;
+   set serveroutput on
+   declare
+     user_exists number;
+   begin
+     select count(*) into user_exists from dba_users where username = 'APEX_PUBLIC_USER';
+     if user_exists = 1 then
+       execute immediate 'alter user APEX_PUBLIC_USER account unlock';
+       execute immediate 'alter user APEX_PUBLIC_USER identified by ApexPassw0rd';
```

Comment:
Oracle security does not permit the use of fixed passwords. This should be randomly generated at runtime and provided to the user via output or log.